src/other/HeatMap.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "../common/CanvasWidget", "simpleheat"], factory);
+    } else {
+        root.other_Heat = factory(root.d3, root.common_CanvasWidget, root.simpleheat);
+    }
+}(this, function (d3, CanvasWidget, simpleheat) {
+    function Heat() {
+        CanvasWidget.call(this);
+    }
+ 
+    Heat.prototype = Object.create(CanvasWidget.prototype);
+    Heat.prototype._class += " other_Heat";
+    Heat.prototype.publish("radius", 25, "number");
+    Heat.prototype.publish("blur", 15, "number");
+    Heat.prototype.publish("max", 1, "number");
```

Comment:
All publish items should have a description as well - and also should define the "tags" in the ext property field (these will be Basic.